docs/batch.rst

```diff
@@ -0,0 +1,118 @@
+################
+Batch operations
+################
+
+.. _batch:
+
+POST /batch
+===========
+
+**Requires an FxA OAuth authentication**
+
+The POST body is a mapping, with the following attributes:
+
+- ``requests``: the list of requests
+- ``defaults``: (*optional*) values in common for all requests
+
+ Each request is a JSON mapping, with the following attribute:
+
+- ``method``: HTTP verb
+- ``path``: URI
+- ``body``: a mapping
+- ``headers``: (*optional*), otherwise take those of batch request
+
+::
+
+    {
+      "defaults": {
+        "method" : "POST",
+        "path" : "/articles",
+        "headers" : {
+          ...
+        }
+      },
+      "requests": [
+        {
+          "body" : {
+            "title": "MoFo",
+            "url" : "http://mozilla.org"
+          }
+        },
+        {
+          "body" : {
+            "title": "MoCo",
+            "url" : "http://mozilla.com"
+          }
+        },
+        {
+          "method" : "PATCH",
+          "path" : "/articles/409",
+          "body" : {
+            "read_position" : 3477
+          }
+        }
+      ]
+    ]
+
+
+The response body is a list of all responses:
+
+::
+
+    {
+      "defaults": {
```

Comment:
I think we should not have defaults for the response. Also it makes sense for the request.